examples/shader/Shader.cpp

```diff
@@ -97,10 +97,7 @@ class WaveBlur : public Effect
         m_text.setPosition({30.f, 20.f});
 
         // Load the shader
-        if (!m_shader.loadFromFile("resources/wave.vert", "resources/blur.frag"))
-            return false;
-
-        return true;
+        return m_shader.loadFromFile("resources/wave.vert", "resources/blur.frag");
```

Comment:
What check caught this? This is a no-brainer check to add and would make for a great PR by itself

Reply:
Likely `readability-simplify-boolean-expr`. Yeah, this PR is starting to get bloated.